recipes-qt/qmllive/qmllive_git.bb

```diff
@@ -0,0 +1,21 @@
+DESCRIPTION = "QmlLive is a live reloader environment for rapid UI development"
+HOMEPAGE = "https://github.com/qtproject/qt-apps-qmllive"
+
+LICENSE = "GPL-3.0"
+LIC_FILES_CHKSUM = " \
+    file://LICENSE.GPL3;md5=75cd0dbc6f2d24e7eeb128ff59b74f4c \
+"
+
+DEPENDS = "qtbase qtdeclarative"
+
+PV = "git${SRCPV}"
```

Comment:
Doesn't it have some more meaningful version?

Reply:
fixed.